cosmo_tester/test_suites/bootstrap_based_tests/inplace_upgrade_test.py

```diff
@@ -0,0 +1,107 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+import pytest
+
+from time import sleep
+from os.path import join
+
+from cosmo_tester.framework.cluster import CloudifyCluster
+from cosmo_tester.framework.examples.hello_world import HelloWorldExample
+
+
+cluster = None
+
+
+@pytest.fixture(scope='module')
+def manager(request, cfy, ssh_key, module_tmpdir, attributes, logger):
```

Comment:
Aha I see.
The second `global` is not needed though (line 68) as you're not reassigning `cluster`, I think exposing the cluster directly from your fixture instead of the manager would be neater than using globals